platform/kubernetes/resource/spec.go

```diff
@@ -0,0 +1,106 @@
+package resource
+
+import (
+	"fmt"
+
+	"github.com/weaveworks/flux/diff"
+)
+
+// Types that daemonsets, deployments, and other things have in
```

Comment:
And also we have the tedious task of going through each K8s type and adding it to this list.